File: laboratories/ppo/exercises/lab05/java/lab05/Car.java
package lab05;

public class Car {
	protected String name;

	public Car(String name) {
		this.name = name;
	}

	public String getName() {
		return this.getType() + " " + this.name;
	}

	protected String getType() {
		return "samochód osobowy";
	}
}

File: laboratories/ppo/exercises/lab05/php/src/Car.php
<?php

namespace Ppo\Lab05;

class Car {
    protected string $name;

    public function __construct(string $name)
    {
        $this->name = $name;
    }

    public function getName(): string
    {
        return $this->getType() . " Car.php" . $this->name;
    }

    protected function getType(): string
    {
        return "samochód osobowy";
    }
}
